libafl_bolts/src/os/unix_signals.rs

```diff
@@ -391,7 +390,7 @@ pub trait Handler {
     /// Handle a signal
     fn handle(&mut self, signal: Signal, info: &mut siginfo_t, _context: Option<&mut ucontext_t>);
     /// Return a list of signals to handle
-    fn signals(&self) -> Vec<Signal>;
+    fn signals(&self) -> &'static [Signal];
```

Comment:
Could we add and remove signals on the fly before? Might have been handy if it worked :D

Reply:
I don't think so, no, this was only ever used to retrieve.